src/rp2040/hardware_structs/include/hardware/structs/clocks.h

```diff
@@ -29,7 +29,7 @@ enum clock_index {
     clk_gpout1,         ///< GPIO Muxing 1
     clk_gpout2,         ///< GPIO Muxing 2
     clk_gpout3,         ///< GPIO Muxing 3
-    clk_ref,            ///< Watchdog and timers reference clock
+    clk_ref,            ///< PLLs, Watchdog and timers reference clock
```

Comment:
This is actually not true. The PLLs are directly connected to the xosc as their reference clock.